### ISSUE TITLE
Bans not removing

### ISSUE THREAD

**Issue by Wild1145** [DarthCraft/DarthCraft#138]
Stupidly the bans are no longer removing themselves as they should. The error is as follows:

[22:41:22] [Server thread/INFO]: [DarthCraft] Removing expired ban: 915c0cf1-ebc3-4a54-b924-fb29f930a35b
[22:41:22] [Server thread/ERROR]: Error occurred while enabling DarthCraft v2.7 (Is it up to date?)
java.lang.NullPointerException
	at net.darthcraft.dcmod.player.Ban.deleteFrom(Ban.java:238) ~[?:?]
	at net.darthcraft.dcmod.addons.BanManager.unban(BanManager.java:231) ~[?:?]
	at net.darthcraft.dcmod.addons.BanManager.removeExpired(BanManager.java:206) ~[?:?]
	at net.darthcraft.dcmod.addons.BanManager.loadBans(BanManager.java:97) ~[?:?]
	at net.darthcraft.dcmod.DarthCraft.onEnable(DarthCraft.java:184) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321) ~[spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:335) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.craftbukkit.v1_8_R2.CraftServer.loadPlugin(CraftServer.java:356) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.craftbukkit.v1_8_R2.CraftServer.enablePlugins(CraftServer.java:316) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.craftbukkit.v1_8_R2.CraftServer.reload(CraftServer.java:746) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.Bukkit.reload(Bukkit.java:534) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.command.defaults.ReloadCommand.execute(ReloadCommand.java:25) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at org.bukkit.craftbukkit.v1_8_R2.CraftServer.dispatchCommand(CraftServer.java:646) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.PlayerConnection.handleCommand(PlayerConnection.java:1133) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.PlayerConnection.a(PlayerConnection.java:968) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.PlayerConnectionUtils$1.run(SourceFile:13) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_31]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_31]
	at net.minecraft.server.v1_8_R2.SystemUtils.a(SourceFile:60) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.MinecraftServer.A(MinecraftServer.java:710) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.DedicatedServer.A(DedicatedServer.java:368) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.MinecraftServer.z(MinecraftServer.java:651) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at net.minecraft.server.v1_8_R2.MinecraftServer.run(MinecraftServer.java:554) [spigot1.8.jar:git-Spigot-dbe012b-61ef214]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_31]

**Comment by Wild1145** [comment]
@JeromSar @WickedGamingUK If either of you two have any idea's, I'd love to hear them now... I've completely ran out of idea's as to what is causing this after 3 hours of looking...

**Comment by JeromSar** [comment]
```
java.lang.NullPointerException
at net.darthcraft.dcmod.player.Ban.deleteFrom(Ban.java:238) ~[?:?]
```
Looks like there's an NPE at line 238:
```java
config.set("uuid." + uuid.toString(), null);
```
This either means that `config` is null, or `uuid` is null. Debug and find out.

**Comment by Wild1145** [comment]
We will be handling banning in Essentials for the time being as the issue just isnt clear, banning will be re-implemented in the 3.0 release.
